cmd/stop.go

```diff
@@ -168,6 +176,38 @@ func runStopSuperNodeSubCommand(ctx context.Context, config *configs.Config) {
 	log.WithContext(ctx).Info("Suppernode stopped successfully")
 }
 
+func runStopSuperNodeRemoteSubCommand(ctx context.Context, config *configs.Config) {
+
+	// Connect to remote
+	client, err := prepareRemoteSession(ctx, config)
+	if err != nil {
+		log.WithContext(ctx).Infof("failed to prepare remote session: %v", err)
+		return
+	}
+	defer client.Close()
+
+	// Execute stop remote supernode
+	log.WithContext(ctx).Info("Executing stop remote supernode...")
+
+	stopOptions := ""
+
+	if len(config.PastelExecDir) > 0 {
+		stopOptions = fmt.Sprintf("--dir %s", config.PastelExecDir)
+	}
+
+	if len(config.WorkingDir) > 0 {
+		stopOptions = fmt.Sprintf("%s --work-dir %s", stopOptions, config.WorkingDir)
+	}
+
+	stopSuperNodeCmd := fmt.Sprintf("%s stop supernode %s", constants.RemotePastelupPath, stopOptions)
+	if err := client.ShellCmd(ctx, stopSuperNodeCmd); err != nil {
+		log.WithContext(ctx).Errorf("Failed to execute stop supernode on remote host - %v", err)
```

Comment:
nit: 
log.WithContext(ctx).WithError(err).Errorf("Failed to execute stop supernode on remote host")